### ISSUE TITLE
Brush does not detect mouse up when releasing outside Bar element

### ISSUE THREAD

**Issue by maryschmidt** [hshoff/vx#613]
Hey, thanks for adding the Brush tool!

When creating a BrushSelection, the mouseUp event is not detected outside of the Bar. I would suggest adding a mouseUp event on the DOM and if dragging is true, behave like mouseUp happened on the Bar.

**Comment by williaster** [comment]
Hey @maryschmidt 👋 thanks for checking out `vx`! I agree proper detection of a `mouseUp`/`onTouchEnd` event would improve the UX of the component (which I would say is still very much a `v0`). 

I likely will not have the bandwidth to add this feature very soon as I'm focusing on improved docs and other `v1` efforts, but I would be happy to review a PR if you / anyone is interested in adding it!

**Comment by maryschmidt** [comment]
#618 If you can take a look at my PR

**Comment by joearasin** [comment]
@williaster It looks like in #667 the window event listeners added in #618 were removed. Was that intentional?

**Comment by williaster** [comment]
@joearasin yes, unfortunately when I actually tested the changes made in #618 , they did not work and so I removed them when also fixing the `initialBrushPosition` logic (which is what #618 turned into). So, still need a working solution for this issue.

**Comment by hshoff** [comment]
Was fixed in https://github.com/airbnb/visx/pull/1164 which added a `useWindowMoveEvents` boolean prop
